tests/test_datetime.rs

```diff
@@ -56,19 +57,14 @@ macro_rules! assert_check_only {
 
 // Because of the relase pool unsoundness reported in https://github.com/PyO3/pyo3/issues/756,
 // we need to stop other threads before calling `py.import()`.
-// TODO(kngwyu): Remove this macro
-macro_rules! lock {
-    () => {
-        let _mutex = std::sync::Mutex::new(());
-        let _lock = _mutex.lock().unwrap();
-    };
-}
+// TODO(kngwyu): Remove this variable
+const MUTEX: RawMutex = RawMutex::INIT;
```

Comment:
I _think_ a `const` is actually not a real variable until the location it's used, so this still creates multiple copies of the mutex.

I think that the suggestion to use `static` on Travis is correct, as then there will only be one shared mutex:

```suggestion
static Mutex: RawMutex = RawMutex::INIT;
```

Reply:
Oops, thank you.